src/decorators/bind.ts

```diff
@@ -40,35 +40,44 @@ export type BindAllDecorator<ValidKeys extends string> = <
 export type BindSingleDecorator = (target: Composite, property: string) => void;
 
 /**
- * A decorator for instance properties on classes extending `Composite`.
+ * A decorator for instance properties of classes extending `Composite`, i.e. a custom component.
+ * It creates a two-way binding between the decorated property and a child (direct or indirect)
+ * of the component. Example:
  *
- * Creates a two-way binding between the decorated property and the property given via the path parameter:
- *
- *  ```
- * ‍@bind(path: '#childId.property')
- * componentProperty: SomeType = initialValue;
+ * ```ts
+ * ‍@bind('#childId.text')
+ * myProp: string = 'foo';
  * ```
  *
- * It's also possible to bind to properties of the object in the component property:
+ * *Notes:*
+ * * *`@bind` behaves like `@property` in most regards.*
```

Comment:
Is this being displayed in an expected way in VSCode? Maybe use another character for bullets, e.g. `-` or `+` to make it easier to read and don't emphasize the bullet point text for simplicity.

Comment also applies to the other occurrences of this pattern.

Reply:
Eh, it just needs to look good in VSC.